x-pack/test/functional/apps/ml/data_visualizer/file_data_visualizer.ts

```diff
@@ -152,6 +153,51 @@ export default function ({ getService }: FtrProviderContext) {
         totalFieldsCount: 3,
         fieldTypeFiltersResultCount: 1,
         fieldNameFiltersResultCount: 1,
+        ingestedDocCount: 13,
+      },
+    },
+    {
+      suiteSuffix: 'with a file with a missing new line char at the end',
+      filePath: path.join(__dirname, 'files_to_import', 'missing_end_of_file_newline.csv'),
+      indexName: 'user-import_2',
```

Comment:
Nit: this index name is already used in the `with a file containing geo field` test. It's be cleaned between the tests so nothing bad happens, but might still be good to use a separate name, e.g. `user-import_3`.

Reply:
Fixed in https://github.com/elastic/kibana/pull/109274/commits/936a18d22787a1d5e2916e3e1c45519ca8834f99